config/locales/cy.yml

```diff
@@ -13,18 +13,26 @@ cy:
     corporate_information: "Gwybodaeth gorfforaethol"
     document_types:
       documents: "Dogfennau"
-      announcements:
+      news_and_communications:
         title: "Cyhoeddiadau"
-        see_all: "Gweld ein holl cyhoeddiadau"
+        see_all:
+          text: "Gweld ein holl cyhoeddiadau"
```

Comment:
If we want to be consistent, this should instead be a translation to say _see all news and communications_

Reply:
We'll need to source a translation for this, something I think we're best doing in a separate PR.

Reply:
Should we put in the request now so that it has chance to happen possibly over Christmas?

Reply:
👍 

I'm not sure what the process is though?